docs/man/antler-proj.1.md

```diff
@@ -18,13 +18,13 @@ antler-proj - Antler Project Management System
 
 # COMMANDS
 
-This is a general help text which prints when antler-proj runs without a command or with key --help.
+This is a general help text which prints when antler-proj runs without a command or with key ---help.
```

Comment:
I think `--help` which was before was better. I think this md file is good for documentation (as well as man). Someone might be mistaken, you need to run `antler-proj ---help` after reading this md file.

Reply:
The problem with `--help` here is the man page will be`-help`. That is wrong. I'm open to a better fix?

Reply:
Can you try solution from Alex: https://github.com/AntelopeIO/cdt/blob/main/docs/man/cdt-ld.1.md?plain=1#L74

Reply:
That seems to work. Fixed in fd60701